backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/InitVdsOnUpCommand.java

```diff
@@ -169,7 +174,8 @@ protected void executeCommand() {
                 haMaintenanceFailed = !hostedEngineHelper.updateHaLocalMaintenanceMode(getVds(), false);
             }
 
-            if (cluster.supportsTrustedService()) {
+            initSucceeded = checkFipsMode();
```

Comment:
i would put it after the trustedService, your call.

Reply:
the attestation service we use [is not supported anymore](https://bugzilla.redhat.com/show_bug.cgi?id=1832406) - so it's insignificant